open-vm-tools-centos/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM centos 
+
+
+
+LABEL summary="The open-vm-tools guest agent" \
+      io.k8s.description="The open-vm-tools agent is providing information about the virtual machine and allows to restart / shutdown the machine via VMware products. This image is intended to be used with virtual machines running Centos Atomic Host." \
+      io.k8s.display-name="open-vm-tools guest agent" \
+      architecture="x86_64" \
+      BZComponent="open-vm-tools" \
+      maintainer="davis phillips <dphillip@redhat.com>"
+
+ENV SYSTEMD_IGNORE_CHROOT=1
+
+RUN yum -y --disablerepo=\* --enablerepo=rhel-7-server-rpms install yum-utils && \
+  yum-config-manager --disable \* && \
+  yum-config-manager --enable rhel-7-server-rpms && \
+  yum clean all
+
+
+RUN yum -y install file open-vm-tools perl open-vm-tools-deploypkg net-tools iproute systemd && \ 
+yum clean all 
+
+ADD service.template config.json.template /exports/
```

Comment:
Here is a [reference](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#add-or-copy) to @yuqi-zhang's note.